helm-charts/FATE/templates/python-spark.yaml

```diff
@@ -19,10 +19,18 @@ metadata:
 {{ include "fate.labels" . | indent 4 }}
 data:
   spark-defaults.conf: |
-    spark.master                      {{  .Values.modules.python.spark.master | default "local[4]"}}
-    #spark.eventLog.enabled           true
-    #spark.eventLog.dir               hdfs://namenode:9000/spark/logs
-    spark.driver.host                 fateflow
+    spark.master                      {{  .Values.modules.python.spark.existingSpark | default "spark://spark-master:7077"}}
```

Comment:
default value is splitting everywhere, could you please unify them in a single place?